.rubocop.yml

```diff
@@ -0,0 +1,58 @@
+inherit_from: .rubocop_todo.yml
+
+require:
+  - rubocop-packaging
+  - rubocop-performance
+
+AllCops:
+  DisplayCopNames: true
+  DisplayStyleGuide: true
+  TargetRubyVersion: 2.4
```

Comment:
Is this the one we do target/should target? Does it match the minimum version? Just asking.

Reply:
This is what we target in Faraday as well. I wanted to keep this fully backwards-compatible in v1.
Once we move to v2 we can obviously increase this since 2.4 and 2.5 are EOL'ed now